brainmodel_utils/metrics/consistency.py

```diff
@@ -225,10 +238,11 @@ def get_linregress_consistency_persphalftrial(
 def get_linregress_consistency(
     source,
     target,
-    map_kwargs,
+    map_kwargs={ "map_type": "identity" },
```

Comment:
Is it possible to not have a default mapping function? Especially since the identity mapping is a passthrough and not a "real" mapping function